recipe/recipe.yaml

```diff
@@ -0,0 +1,84 @@
+schema_version: 1
+
+context:
+  version: 2025.5.1
+  python_min: 3.10
+
+package:
+  name: dask
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/d/dask/dask-${{ version }}.tar.gz
+  sha256: 979d9536549de0e463f4cab8a8c66c3a2ef55791cd740d07d9bf58fab1d1076a
+
+build:
+  number: 1
+  noarch: python
+
+requirements:
+  host:
+    - python ${{ python_min }}.*
+    - dask-core 2025.5.1.*
+    - distributed 2025.5.1.*
```

Comment:
`dask`, `dask-core`, and `distributed` are all pinned tightly. Previously we had

```
  host:
    - python {{ python_min }}
    - dask-core {{ version }}
    - distributed {{ version }}
```

here to ensure we always have that tight pinning automatically. Can we keep that same functionality with the v1 spec?

Reply:
ah, thanks for the catch